lib/token-assert.js

```diff
@@ -96,11 +96,14 @@ TokenAssert.prototype.sameLine = function(options) {
 };
 
 /**
- * Requires tokens to be on different lines.
+ * Requires tokens to be on different lines. If atLeast, atMost, and exactly, aren't specified, defaults to at least 1.
  *
  * @param {Object} options.token
  * @param {Object} options.nextToken
  * @param {Object} [options.message]
+ * @param {Object} [options.atLeast] At least how many lines the tokens are apart
+ * @param {Object} [options.atMost] At most how many lines the tokens are apart
+ * @param {object} [options.exactly] Exactly how many lines the tokens are apart
```

Comment:
I saw https://github.com/jscs-dev/node-jscs/pull/1014#discussion_r24298315 which used `lines` and see that the current PR uses `exactly`. But I can't seem to find when the discussion about moving to `exactly``/atLeast`/`atMost`. Can you link me?